ruleset/sca/rhel/7/cis_rhel7_linux.yml

```diff
@@ -1089,7 +1089,7 @@ checks:
       - tsc: ["CC5.2"]
     condition: all
```

Comment:
Please, change this `condition` to `none` to make the check pass when `xorg-x11-server` is not installed.